bigquery_etl/cli/query.py

```diff
@@ -1488,7 +1520,12 @@ def _update_query_schema(
                 existing_schema.to_yaml_file(existing_schema_path)
 
     # replace temporary table references
-    sql_content = query_file_path.read_text()
+    sql_content = render_template(
```

Comment:
I had to add this so the `bqetl query schema update` worked for templated queries.